src/app/components/domain/domain.component.html

```diff
@@ -1,26 +1,23 @@
 <app-header>
   <app-domain-check></app-domain-check>
 </app-header>
-<div class="container-fluid">
-  <section class="row about">
+<aside class="container-fluid about">
+  <article>
     <div class="container">
-    <h4>{{'About Zonemaster'|translate}}</h4>
-      <p>
-        {{'ABOUT_ZONEMASTER_TEXT'|translate}}
+    <h4 i18n>About Zonemaster</h4>
+      <p i18n>
+        Give your domain a complete physical! Zonemaster helps you to control how your website is doing. As a bonus, you also get a better understanding of how the domain name system, DNS, works. When a domain (also called zone) is sent to Zonemaster, the program investigates the state of the domain from beginning to end. This is done through Zonemaster examining DNS from the root (.) to the TLD (top-level domain, for example, .se), and then finally through the DNS servers that contain information about the specified domain (for example, zonemaster.se). Zonemaster is also able to perform many other tests, for example, checking the DNSSEC signatures, that different hosts can be accessed and that the IP addresses are valid. This is all to make sure that your domain runs as good as possible.
```

Comment:
Can these long lines be broken up into 80-character lines to make it easier to read and review?

Reply:
Fixed